test/test_client.c

```diff
@@ -11,7 +11,7 @@
 
```

Comment:
Let's remove this file, as this is for testing only.
We can use `gnutls-cli` to test the server.